### DISCUSSION TITLE
Not able to see data from new index

### DISCUSSION THREAD

**Discussion by pd98** [Security-Onion-Solutions/securityonion#5420] [Unsupported Versions] [answered]
Hi,

I created a new index in elasticsearch through logstash in which i am ingesting CEF logs. I am able to view the data in kibana but not able to see it in security onion console. Please note that i am using a separate logstash and not the logstash that is installed with security onion.
Details : 
1. Security Onion Version : 2.3.61
2. Install source : ISO
3. Install type : standalone
4. so-status : all ok

Any help would be appreciated. Thanks

**Comment by dougburks** [comment]
What is your index named?

Have you tried naming it with the `so-` prefix?

**Reply by pd98** [reply]
currently my index is named in the following format logstash-xxxx.
I tried creating a new index with so-logstash-xxxx but I wasn't able to insert any data into this index since it kept showing errors related to mapping objects.
I have kept the manage template as false in my logstash configuration file.

**Reply by dougburks** [reply]
SOC's `DEFAULT_INDEX` setting defaults to `*:so-*`:
https://github.com/Security-Onion-Solutions/securityonion-soc/blob/1c8bf0358bcb8c6c9906f3841c34cbc81f987d57/server/modules/elastic/elastic.go#L23

So another option might be to try to override this setting in `soc.json`.

**Reply by pd98** [reply]
Hi,
I can see 2 soc.json files 
1 ./so/saltstack/default/salt/soc/files/soc/soc.json
2 ./so/conf/soc/soc.json

I searched if i could find any default_index variable but no such constant was set in both these files.
can you please let me know in which of these i should make changes and what i should add?

**Reply by dougburks** [reply]
You should be able to copy `/opt/so/saltstack/default/salt/soc/files/soc/soc.json` to `/opt/so/saltstack/local/salt/soc/files/soc/soc.json` and then make your changes there. 

Don't forget to then restart SOC with:
```
sudo so-soc-restart
```

**Reply by pd98** [reply]
Hi @dougburks,

I checked the file that you mentioned but there is no option to set the index pattern. I am attaching a screenshot of the soc.json file.
![image](https://user-images.githubusercontent.com/33838706/135397190-936f2036-876d-4d86-ac48-ea6f2fab7262.png)

**Comment by pd98** [comment, marked as answer]
I found the solution to this.

Renaming the index to use the prefix "so-" works however if you face any issue with field mappings, use mutate to remove that particular field.
`mutate {
      remove_field => [ "host" ]
    }`
In my case removing the "host" field worked and logstash did not throw any other field mapping errors.
